app/test_split_integer.py

```diff
@@ -1,21 +1,27 @@
+import pytest
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(8, 4)) == 8
+    assert sum(split_integer(17, 3)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(8, 4) == [2, 2, 2, 2]
+    assert split_integer(15, 5) == [3, 3, 3, 3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(10, 1) == [10]
+    assert split_integer(7, 1) == [7]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(17, 4)
+    assert result == sorted(result)
```

Comment:
The test `test_parts_should_be_sorted_when_they_are_not_equal` checks if the result is sorted. This might not be necessary if the sorting of parts is not a requirement of the function. Consider removing this test if the order of parts is not important.